dev-infrastructure/configurations/mvp-svc-cluster.bicepparam

```diff
@@ -0,0 +1,41 @@
+using '../templates/svc-cluster.bicep'
+
+param kubernetesVersion = '1.29.2'
+param istioVersion = 'asm-1-20'
+param vnetAddressPrefix = '10.128.0.0/14'
+param subnetPrefix = '10.128.8.0/21'
+param podSubnetPrefix = '10.128.64.0/18'
+param enablePrivateCluster = false
+param persist = true
+param aksKeyVaultName = 'aks-kv-aro-hcp-dev-sc'
+param disableLocalAuth = false
```

Comment:
what's the plan to switch this?  

also what's the `disableLocalAuth` apply to?  Cosmosdb, postgres, etc.  Do we need to split them out?

Reply:
applies to cosmos only / the postgres module is configured with entra auth only